src/tracks/playabletrack/wavetrack/ui/WaveformView.cpp

```diff
@@ -353,6 +355,9 @@ void DrawMinMaxRMS(
    long pixAnimOffset = (long)fabs((double)(wxDateTime::Now().GetTicks() * -10)) +
       wxDateTime::Now().GetMillisecond() / 100; //10 pixels a second
 
+   const auto ms = wxDateTime::Now().GetMillisecond();
+   const auto ticks = (long)fabs((double)(wxDateTime::Now().GetTicks() * -10));
```

Comment:
unused?

Reply:
It will be removed in the next PR anyway, but not quite clear how it ended up here anyway